zoo-css-challenge/style.css

```
@@ -188,8 +229,14 @@ footer {
   color: #ff7f11;
 }
 
+.membership p{
+  max-width: 700px;
+  margin-left:300px ;
```

```suggestion
  margin: auto;
```
We can use `margin-auto` which centres the element. `auto` distributes the space left and right equally